### ISSUE TITLE
UI inconsistent between "new task" and "edit task" views

### ISSUE THREAD

**Issue by paullemmens** [mpcjanssen/simpletask-android#550]
This might be a deliberate choice, nevertheless, in terms of UI polish it would be prettier to make the UI more uniform. I personally prefer the more textual versions of the icons/buttons.

![screenshot_20160831-093606](https://cloud.githubusercontent.com/assets/5063702/18120511/361c74f4-6f60-11e6-9735-27ac86ff6f75.png) ![screenshot_20160831-093640](https://cloud.githubusercontent.com/assets/5063702/18120512/3620b0b4-6f60-11e6-85c6-6652f0d00710.png)

**Comment by smichel17** [comment]
Duplicate of #443, I'll reply there.

**Comment by smichel17** [comment]
I am always curious about more people's answers to #458, if you have some time.

**Comment by paullemmens** [comment]
Done so

**Comment by smichel17** [comment]
You'll be happy that buttons for due and threshold
 date will be added in the next release. They're in master right now if you want to compile yourself; the release is waiting on a little more testing to make sure the changes work on older and smaller devices.

**Comment by mpcjanssen** [comment]
@smichel17 changes look good on a Nexus S emulator.

**Comment by smichel17** [comment]
Closed by 872010b.
